core/build/webpack.ssr.config.ts

```diff
@@ -0,0 +1,13 @@
+import merge from 'webpack-merge';
+import base from './webpack.base.config';
+import nodeExternals from 'webpack-node-externals';
```

Comment:
I've added it as it's required for webpack, in this case, to provide express but I can try to find solid solution instead of this workaround.